### PR TITLE
Interface doc fix

### PR DESCRIPTION
**Pull request by achalpandey**

### CODE REVIEW

File: tigon-docs/developer-guide/source/developer.rst [hunk 1]
```diff
@@ -578,16 +578,26 @@ schema:
...
 increasing or decreasing might be required for certain SQL queries; for example, "GROUP BY
 *increasingField*".
 
 Once one or more ``StreamSchemas`` are created, they are added as an input using the
-``addJSONInput`` method. This method takes the name of the input stream and the schema of
-the stream. Once the inputs streams have been added, one or more SQL queries can be
-defined using an ``addQuery`` method. The ``addQuery`` method takes the name of the query
-and the SQL statement.
+``addJSONInput`` method. This method takes the name of the input stream (interface) and
+the associated schema object. Once the inputs streams have been added, one
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: Should this be "Once the input stream"? Singular or plural?

File: tigon-docs/developer-guide/source/developer.rst [hunk 2]
```diff
@@ -578,16 +578,26 @@ schema:
...
+or more SQL queries can be defined using an ``addQuery`` method. The ``addQuery`` method
+takes the name of the query and the SQL statement.
+
+An interface represents the end-point at which the data stream is ingested and this data
+will be interpreted by all the schemas associated with it. The ``FROM`` clause in
+Tigon SQL queries should be followed by a data source name that prescribes to the
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: TigonSQL queries should be followed by a data source name that follows the format of *interfaceName.schemaName*.

File: tigon-docs/developer-guide/source/developer.rst [hunk 3]
```diff
@@ -578,16 +578,26 @@ schema:
...
+is monotonically increasing or decreasing. A field with this attribute set to
 increasing or decreasing might be required for certain SQL queries; for example, "GROUP BY
 *increasingField*".
 
 Once one or more ``StreamSchemas`` are created, they are added as an input using the
-``addJSONInput`` method. This method takes the name of the input stream and the schema of
-the stream. Once the inputs streams have been added, one or more SQL queries can be
-defined using an ``addQuery`` method. The ``addQuery`` method takes the name of the query
-and the SQL statement.
+``addJSONInput`` method. This method takes the name of the input stream (interface) and
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: This method takes the name of an input stream (an interface)...

File: tigon-docs/developer-guide/source/developer.rst (hunk 1 above)
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: Or should this be: "Once all the input streams have been added, one..."

File: tigon-docs/developer-guide/source/developer.rst [hunk 4]
```diff
@@ -578,16 +578,26 @@ schema:
...
+the associated schema object. Once the inputs streams have been added, one
+or more SQL queries can be defined using an ``addQuery`` method. The ``addQuery`` method
+takes the name of the query and the SQL statement.
+
+An interface represents the end-point at which the data stream is ingested and this data
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: An interface represents the end-point [from?] which the data stream is ingested

File: tigon-docs/developer-guide/source/developer.rst [hunk 5]
```diff
@@ -578,16 +578,26 @@ schema:
...
+will be interpreted by all the schemas associated with it. The ``FROM`` clause in
```
Comment (gokulavasan) [inline tigon-docs/developer-guide/source/developer.rst]: Should we mention that currently there can be only one schema associated with an interface?

File: tigon-docs/developer-guide/source/developer.rst [hunk 6]
```diff
@@ -578,16 +578,29 @@ schema:
...
-defined using an ``addQuery`` method. The ``addQuery`` method takes the name of the query
-and the SQL statement.
+``addJSONInput`` method. This method takes the name of an input stream (an interface) and
+the associated schema object. Once all the input streams have been added, one
+or more SQL queries can be defined using an ``addQuery`` method. The ``addQuery`` method
+takes the name of the query and the SQL statement.
+
+Conceptually, an interface represents the TCP end-point at which the data stream is
+ingested and this data is interpreted by all the schemas associated with this
+end-point. Please note, for now Tigon only supports one schema per interface. This
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: Currently, Tigon supports only one schema per interface. This restriction may be removed in a subsequent Tigon release.

File: tigon-docs/developer-guide/source/developer.rst [hunk 7]
```diff
@@ -578,16 +578,29 @@ schema:
...
+restriction may be removed in subsequent Tigon releases.
+
+The ``FROM`` clause in Tigon SQL queries should be followed by a data source name that
+follows the format of *interfaceName.schemaName*. In the example below ``intInput`` is
+the schema name and ``inputStream`` is the interface name. To access ``intInput`` the
+data source is referenced as ``inputStream.intInput``.
+
+For more information about interface and interface sets please refer to the Tigon SQL
+User Manual.
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: For more information on interface and interface sets, please refer to the :doc:`Tigon SQL
+User Manual. <apis/index>`

File: tigon-docs/developer-guide/source/developer.rst [hunk 8]
```diff
@@ -578,16 +578,29 @@ schema:
...
-the stream. Once the inputs streams have been added, one or more SQL queries can be
-defined using an ``addQuery`` method. The ``addQuery`` method takes the name of the query
-and the SQL statement.
+``addJSONInput`` method. This method takes the name of an input stream (an interface) and
+the associated schema object. Once all the input streams have been added, one
+or more SQL queries can be defined using an ``addQuery`` method. The ``addQuery`` method
+takes the name of the query and the SQL statement.
+
+Conceptually, an interface represents the TCP end-point at which the data stream is
+ingested and this data is interpreted by all the schemas associated with this
```
Comment (achalpandey) [inline tigon-docs/developer-guide/source/developer.rst:593]: "this data" refers to the data in the data stream.

File: tigon-docs/developer-guide/source/developer.rst [hunk 9]
```diff
@@ -578,16 +578,29 @@ schema:
...
+end-point.Currently, Tigon supports only one schema per interface. This restriction
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: Insert a space after the period: "end-point. Currently,"

File: tigon-docs/developer-guide/source/developer.rst [hunk 10]
```diff
@@ -578,16 +578,29 @@ schema:
...
+may be removed in a subsequent Tigon release
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: Add a trailing period.

File: tigon-docs/developer-guide/source/developer.rst [hunk 11]
```diff
@@ -578,16 +578,29 @@ schema:
...
+end-point.Currently, Tigon supports only one schema per interface. This restriction
+may be removed in a subsequent Tigon release 
+
+The ``FROM`` clause in Tigon SQL queries should be followed by a data source name that
+follows the format of *interfaceName.schemaName*. In the example below ``intInput`` is
+the schema name and ``inputStream`` is the interface name. To access ``intInput`` the
+data source is referenced as ``inputStream.intInput``.
+
+For more information on interface and interface sets, please refer to the :doc:`Tigon SQL
++User Manual. <apis/index>`
```
Comment (awholegunch) [inline tigon-docs/developer-guide/source/developer.rst]: this should be "Tigon SQL User Manual"; an extra "+" slipped in there.

**Comment by awholegunch** [comment]
some minor formatting nits, otherwise LGTM...

**Pull request merged**